app/components/common/pageHeaderBreadcrumbs/pageHeaderBreadcrumbs.js

```diff
@@ -0,0 +1,9 @@
+(function() {
+  'use strict';
+
+  var pageHeaderBreadcrumbsDirective = require('./pageHeaderBreadcrumbsDirective');
+
+  module.exports = angular.module('pageHeaderBreadcrumbsModule', ['ui.router', 'ui.bootstrap', 'ncy-angular-breadcrumb'])
+    .directive('pageHeaderBreadcrumbs', pageHeaderBreadcrumbsDirective);
+
+})();
```

Comment:
Did you see my comments from your last [PR](https://github.com/caciviclab/disclosure-frontend-alpha/pull/25)? You don't need the closure wrapper.